plugins/org.jboss.tools.jst.web.ui/src/org/jboss/tools/jst/web/ui/internal/editor/contentassist/AutoContentAssistantProposal.java

```diff
@@ -141,7 +141,7 @@ public void apply(ITextViewer viewer, char trigger, int stateMask, int offset) {
 		}
 		
 	    super.apply(viewer, trigger, stateMask, offset + diff);
-	    if(autoContentAssistant) {
+	    if(autoContentAssistant && (originalProposal==null || originalProposal.isAutoActivationContentAssistantAfterApplication())) {

```

Comment:
It is related. We didn't use this API before. Now we need it for angularjs to correct the selection after inserting angular expression and for avoiding invoking content assist automaticly one more time after inserting an angular expression. For example, when you select the <image> proposal then the <image src="|"> will be inserted and content assist will be invoked automatically one more time for src="|" attribute. We should avoid this auto content assist for {{XYZ}} until any '.' typed: {{XYZ.|}